src/vs/workbench/contrib/timeline/browser/timelinePane.ts

```diff
@@ -328,7 +328,7 @@ export class TimelinePane extends ViewPane {
 		let pageSize = this.configurationService.getValue<number | null | undefined>('timeline.pageSize');
 		if (pageSize === undefined || pageSize === null) {
 			// If we are paging when scrolling, then add an extra item to the end to make sure the "Load more" item is out of view
-			pageSize = Math.max(20, Math.floor((this.tree.renderHeight / ItemHeight) + (this.pageOnScroll ? 1 : -1)));
+			pageSize = Math.max(20, Math.floor((this.tree?.renderHeight ?? 0 / ItemHeight) + (this.pageOnScroll ? 1 : -1)));
```

Comment:
Should the `!` be removed here too?

https://insiders.vscode.dev/github/microsoft/vscode/blob/d1c586beeea0e473423217c00c0dc2bb24dfff1a/src/vs/workbench/contrib/timeline/browser/timelinePane.ts#L238-L239

Reply:
That would be a much larger change which I didn't want to do at the moment which is why I held off as described by the PR description. In the end though, yes I will probably change that as it provides a false sense of security